docs/src/openapi-v1.yaml

```diff
@@ -1793,6 +1795,23 @@ components:
         trieIndex:
           type: string
           format: unsignedInteger
+    GenericExtrinsicEra:
+      type: object
+      description: |
+        The return value for eraInfo can either be `mortalEra`, or `immortalEra` and is represented as an enum in substrate. `immortalEra` meaning 
+        the transaction is valid forever. `mortalEra` consists of a tuple containing a period and phase.
+        ex: `"{"mortalEra": ["64", "11"]}"`. The Period is the period of validity from the block hash found in the signing material. 
+        The Phase is the period that this transaction's lifetime begins (and, importantly, 
+        implies which block hash is included in the signature material).
+      properties:
+        mortalEra: 
+          type: string
+          nullable: true
+        immortalEra:
+          type: string
+          nullable: true
+          default: ['0x00']
```

Comment:
If I understand this correctly, this is the only possible value for the `immortal` case? So rather than a default, it's just the hard coded value that must be there for Reasons™? If that is correct, can we get rid of it somehow, and have just `immortalEra`?

Reply:
One way to represent the return types I guess could be:

```
era: {
   type: "immortal"
}
```

or

```
era: {
   type: "mortal",
   period: 64,
   phase: 12
}
```

(with a typescript type like `{ type: "immortal" } | { type: "mortal", period: number, phase: number }`)

Reply:
> A description explaining that the value is hardcoded or will always be '0x00' would possibly suffice?

That's ok for this PR. Perhaps add an issue to investigate James' suggestion?

Reply:
Sweet, so I added a concise description for `immortal` -> `Hardcoded constant '0x00'`. I do like the idea of the return type being as @jsdw put it. The one reason I like it is the natural response from substrate has `mortal` being iterable, and `immortal` just being a string. Having different data structures in the same response field is a little odd for our circumstances. 

I do think for now it might be wise to keep the response as close to substrate as possible. The idea being if anything were to change in substrate (low likely hood), the little logic there is for getting `era` wont change.